vllm/entrypoints/openai/serving_chat.py

```diff
@@ -75,6 +75,11 @@ def __init__(self,
             try:
                 self.tool_parser = ToolParserManager.get_tool_parser(
                     tool_parser)
+                if (self.tool_parser.__name__ == "PythonicToolParser" and
```

Comment:
Perhaps it would be better to match against `tool_parser` variable instead?

Reply:
:man_facepalming: Yes, of course. Done.